localstack-core/localstack/testing/test_config.py

```diff
@@ -0,0 +1,18 @@
+import os
```

Comment:
Calling this `test_config` is a bit of a misnomer as it looks like a test file. 

Consider simply calling it `localstack.testing.config`

Reply:
Ah, did not notice that, good point! Will change it!